redisgraph/execution_plan.py

```diff
@@ -0,0 +1,61 @@
+class Operation:
+    def __init__(self, name, args=None):
+        self.name = name
+        self.args = args
+        self.children = []
+
+    def append_child(self, child):
+        self.children.append(child)
+        return self
+
+    def __eq__(self, o: object) -> bool:
```

Comment:
Can we dictate `O`'s type ? if so it should be of type `Operation`

Reply:
no we can't